snuba/datasets/discover.py

```diff
@@ -85,22 +86,54 @@ def get_data_source(self):
         return None
 
     def get_columns(self) -> ColumnSet:
-        common = ColumnSet([])  # TODO: Fill out
+        common = ColumnSet([
+            ('event_id', FixedString(32)),
+            ('project_id', UInt(64)),
+            ('timestamp', DateTime()),
+            ('deleted', UInt(8)),
+            ('retention_days', UInt(16)),
+            ('platform', Nullable(String())),
+            ('sentry:release', Nullable(String())),
+            ('sentry:dist', Nullable(String())),
+            ('sentry:user', Nullable(String())),
```

Comment:
Why these names over `release`, `dist` and `user`? Should `environment` be here too?

Reply:
Yes, added `environment`. I picked these names since they are the ones in the Events dataset and those are the names Sentry is always passing. Introducing completely new names while we do the migration is going to be complicated, and Sentry maps them to other public names anyway.